This patch fixes the install path for the kernel module and disables
depmod.

-- 
Jonas Karlsson

--- kqemu-1.3.0pre11/install.sh	2007-02-06 22:02:00.000000000 +0100
+++ kqemu-1.3.0pre11/install.sh.new	2007-09-26 00:07:50.000000000 +0200
@@ -10,9 +10,9 @@
 fi
 
 # Find kernel install path
-kernel_path="/lib/modules/`uname -r`"
+kernel_path="@%Compile_goboModules%@/`uname -r`"
 
 mkdir -p "$kernel_path/misc"
 cp "$module" "$kernel_path/misc"
 
-/sbin/depmod -a
+# /sbin/depmod -a
